src/ResourceManager/Compute/Commands.Compute/Manual/NewAzureRmVmssConfigCommand.cs

```diff
@@ -0,0 +1,38 @@
+// 
+// Copyright (c) Microsoft and contributors.  All rights reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// 
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// 
+
+// Warning: This code was generated by a tool.
+// 
+// Changes to this file may cause incorrect behavior and will be lost if the
+// code is regenerated.
+
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.Compute.Automation
+{
+    public partial class NewAzureRmVmssConfigCommand : Microsoft.Azure.Commands.ResourceManager.Common.AzureRMCmdlet
+    {
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = false)]
+        [Obsolete("This parameter is obsolete.  Please use IdentityType parameter instead.", false)]
```

Comment:
@hyonholee would you mind documenting this change in the [`upcoming-breaking-changes.md`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/Compute/documentation/upcoming-breaking-changes.md) file?

Also, would you mind changing the text of the warning to something like the following:

`New-AzureRmVmssConfig: The parameter AssignIdentity will be removed in an upcoming release. Please use the IdentityType parameter instead.`